src/app/style/abstract/_icons.scss

```diff
@@ -48,6 +48,16 @@
             top-left-radius: 2px;
             top-right-radius: 2px;
         };
+
+        @include largest-screen {
+            @-moz-document url-prefix() {
+                box-shadow:
+                    6px 3px 0 -1px var(--header-color),
+                    -6px 3px 0 -1px var(--header-color),
+                    -2px 3px 0 -1px var(--header-color),
+                    2px 3px 0 -1px var(--header-color);
+            }
```

Comment:
I believe we have autoprefixer, which should handle prefixes automatically. Please check if it does and if it does - remove the explicit prefix notation here.